aiida/engine/processes/builder.py

```diff
@@ -188,4 +190,5 @@ def __init__(self, process_class):
 
     @property
     def process_class(self):
+        """Straighforward wrapper for the process_class"""
```

Comment:
Not really a useful docstring I think, how about
```suggestion
        """Return the process class for which this builder is constructed."""
```